### PR TITLE
Fix index for Polish books

### PR DESCRIPTION
**Pull request by wilkus27**
Issue: https://github.com/openstax/kitchen/issues/442

### CODE REVIEW

**Review by ajluk — APPROVED** [review]

File: lib/kitchen/patches/i18n.rb [hunk 1]
```diff
@@ -4,6 +4,10 @@
 
 # rubocop:disable Style/Documentation
 module I18n
+  def self.group_by(character)
```
Comment (kswanson33) [inline lib/kitchen/patches/i18n.rb:7]: `group_by` as a name makes sense in the context of indexes, but this function doesn't group anything, it just decides what character to return based on locale. Maybe a better name would be something like `transliterate_for_index` or `character_to_group`, or something I haven't thought of! lmk what you think :)

File: spec/helpers/stub_helpers.rb [hunk 2]
```diff
@@ -1,17 +1,19 @@
...
     I18n.config.available_locales = %i[test en es pl]
     allow_any_instance_of(I18n::Config).to receive(:backend).and_return(
       I18n::Backend::Simple.new.tap do |backend|
-        backend.store_translations 'test', hash
+        backend.store_translations language, hash
       end
     )
 
-    assign_i18n_dot_locale(:test)
+    assign_i18n_dot_locale(language)
```
Comment (kswanson33) [inline spec/helpers/stub_helpers.rb:16]: nice

**Review by kswanson33 — CHANGES_REQUESTED** [review]
I like what you've done here! One naming-related comment, but otherwise it looks great to me 🎉

File: lib/kitchen/patches/i18n.rb (hunk 1 above)
Comment (wilkus27) [reply lib/kitchen/patches/i18n.rb]: Oh good point :+1: I didn't think about it. I like very much second proposition. It's more descriptive. I made a fix for that.

**Pull request merged**